TicTacToe/Board.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace TicTacToe
+{
+    public class Board
+    {
+        public char[] GameBoard { get; set; }
+
+        public Board(int size)
+        {
+            GameBoard = new char[size * size];
+            InitializeBoard();
+        }
+
+        public void InitializeBoard()
+        {
+            for (int i = 0; i < GameBoard.Length; i++)
+            {
+                MakeMove(' ', i);
+            }
+        }
+        public void MakeMove(char symbol, int position)
+        {
+            GameBoard[position] = symbol;
+        }
+
+        public int GetAvailableMoves()
+        {
+            int counter = 0;
+            for(int i = 0; i < GameBoard.Length; i++)
```

Comment:
I'm not as familiar with C#, but I wonder if you can replace the for loop with [Array.FindAll](https://docs.microsoft.com/en-us/dotnet/api/system.array.findall?redirectedfrom=MSDN&view=netframework-4.8#System_Array_FindAll__1___0___System_Predicate___0__).

Reply:
okay